### DISCUSSION TITLE
v.0.8.0

### DISCUSSION THREAD

**Discussion by DavidLazarescu** [Librum-Reader/Librum#46] [General]
- Added highlighting
- Improved feedback
- Added drag & drop for the library
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Librum-Reader/Librum/releases/tag/v.0.8.0'>v.0.8.0</a>.</em>

**Comment by alvi-khan** [comment]
Hey @DavidLazarescu! Super excited about this project. It's amazing how fast you're working!

Just wanted to give you a heads up in case you haven't noticed, but your repo is a little big (1GB+ including the .git folder). This makes it a little difficult to clone. You should consider taking a look into this when you can.

**Reply by DavidLazarescu** [reply]
Hey, thanks for the feedback! We have a dependency on mupdf which is pretty big since it in turn has a lot of dependencies on other graphic and text related libraries for the rendering. I suppose that is what you mean, right?

**Reply by alvi-khan** [reply]
Yes, that's the one. Sorry I don't really have any suggestions on how to improve the situation though. :(

**Comment by lonanl** [comment]
It looks promising! You are great fellows. The application is already ready to use. 
I hope this project will finally allow you to finally read e-books on computers.

**Reply by DavidLazarescu** [reply]
Thanks, we hope you like it. Many other great features will come very soon!

Also, we are always happy to get some feedback, if you run into any problems or have improvement proposals, please reach out to us!
